packages/financial-templates-lib/src/logger/PagerDutyTransportV2.ts

```diff
@@ -0,0 +1,49 @@
+// This transport enables winston logging to send messages to pager duty v2 api.
+import Transport from "winston-transport";
+import { event } from "@pagerduty/pdjs";
```

Comment:
The previous implementation uses the depricated `node-pagerduty`: https://github.com/UMAprotocol/protocol/blob/9fabd6cd7d2f1d0508199f779f64957f950c5f42/packages/financial-templates-lib/src/logger/PagerDutyTransport.ts#L7

this old package should likely be removed.

Reply:
im not ready to remove this, as i dont know how well the v2 api will work, theres still a lot of unknowns until i can get some real world data working